apps/prairielearn/elements/pl-integer-input/pl-integer-input.mustache

```diff
@@ -18,6 +18,7 @@
             {{#raw_submitted_answer}}value="{{raw_submitted_answer}}"{{/raw_submitted_answer}}
             aria-describedby="pl-integer-input-{{uuid}}-label pl-integer-input-{{uuid}}-suffix"
             placeholder="{{placeholder}}"
+            aria-labelledby="pl-integer-input-{{uuid}}-label"
```

Comment:
As discussed on Zoom, can you test how this works when the label contains MathJax?

If we're using this as a label, it should be removed from `aria-describedby` above. Ditto for other elements.

Reply:
Tested with `workshop/Lesson3_example2_v1`, `workshop/Lesson3_example2_v2`, `workshop/Lesson3_example4_v1` in the example course and `positionTimeGraph` in the test course. These questions all have labels that contain MathJax and seemed to work correctly with a screen reader. 

Also, removed the label from `aria-describedby` here and elsewhere in [b2ddc80](https://github.com/PrairieLearn/PrairieLearn/pull/10514/commits/b2ddc80c9af2e30a4fa6ea94027150ce2ac8b3ba).

Reply:
Nice! Out of curiosity, what does it get announced as? I suppose I could always just test for myself 🙂

Reply:
Let's continue the conversation here: https://github.com/PrairieLearn/PrairieLearn/pull/10514#discussion_r1737517389